src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -835,6 +826,20 @@ export default {
 
 			return displayName
 		},
+
+		getButtonsBarPositionStyle() {
+			if (!this.showMessageButtonsBar) {
+				return
+			}
+
+			// Check if the bottom of messageButtonsBar is outside visible area (36px from message bottom)
+			const messageListBottom = this.$parent.$parent.$el.getBoundingClientRect().bottom
+			const buttonsBarSupposedBottom = this.$refs.message.getBoundingClientRect().bottom + 36
+
+			if (messageListBottom < buttonsBarSupposedBottom) {
+				return `bottom: calc(100% - ${this.$refs.messageInfo.offsetTop}px)`
+			}
```

Comment:
The solution based on `this.$parent.$parent.$el` in a Vue component depends on other components' implementation in a very implicit way. Maybe we can achieve it in a more explicit solution?

Reply:
Also, "36" feels like a magic number here to me. If I understand correctly, it is calculated from the current value of the `$clickable-area` SASS variable. Still, it doesn't depend on the variable value (changing the SASS variable value would require updating this value in the code). If it is equal to the `.message-buttons-bar` bottom property, could we get the element's current value? Or the `--default-clickable-area` CSS variable if it is the same thing.